Detectors/ZDC/workflow/src/ZDCDataReaderDPLSpec.cxx

```diff
@@ -33,6 +33,21 @@ void ZDCDataReaderDPLSpec::init(InitContext& ic)
 
 void ZDCDataReaderDPLSpec::run(ProcessingContext& pc)
 {
+  // if we see requested data type input with 0xDEADBEEF subspec and 0 payload this means that the "delayed message"
+  // mechanism created it in absence of real data from upstream. Processor should send empty output to not block the workflow
+  {
+    std::vector<InputSpec> dummy{InputSpec{"dummy", ConcreteDataMatcher{o2::header::gDataOriginFT0, o2::header::gDataDescriptionRawData, 0xDEADBEEF}}};
+    for (const auto& ref : InputRecordWalker(pc.inputs(), dummy)) {
+      const auto dh = o2::framework::DataRefUtils::getHeader<o2::header::DataHeader*>(ref);
+      if (dh->payloadSize == 0) {
+        LOGP(WARNING, "Found input [{}/{}/{:#x}] TF#{} 1st_orbit:{} Payload {} : assuming no payload for all links in this TF",
+             dh->dataOrigin.str, dh->dataDescription.str, dh->subSpecification, dh->tfCounter, dh->firstTForbit, dh->payloadSize);
+        mRawReader.makeSnapshot(pc); // send empty output
```

Comment:
Are you sure this will not use whatever was left in the mRawReader from the previous TF?
Perhaps one should move `mRawReader.clear();` before the check?